torch/csrc/functorch/init.cpp

```diff
@@ -449,7 +449,6 @@ void initFuncTorchBindings(PyObject* module) {
   m.def("dump_tensor", &dump_tensor, "dump_tensor");
   m.def("reshape_dim_into", &at::functorch::reshape_dim_into);
   m.def("reshape_dim_outof", &at::functorch::reshape_dim_outof);
-  m.def("are_transforms_active", &at::functorch::areTransformsActive);
```

Comment:
Through the other refactors did we end up just not using this in the Python? Somehow I remember it being used somewhere but it's probably fine if all the tests are passing